app/src/main/java/com/paranoidandroid/journey/legplanner/activities/PlannerActivity.java

```diff
@@ -383,12 +383,10 @@ public void addFromBookmarksPressed(View v) {
     public void addCustomActivityPressed(View v) {
         Day day = getDayPlannerFragment().getSelectedDay();
         Destination dest = day.getLeg().getDestination();
-        FragmentManager fragmentManager = getSupportFragmentManager();
+        FragmentManager fragmentManager = getFragmentManager();
```

Comment:
Is it safe to use this when inheriting from AppCompatActivity? I remember Nathan saying it could cause problems to use getFragmentManager() in this case.